test/lexical-mdx/tests/default.test.ts

```diff
@@ -400,6 +400,19 @@ there4
       },
     },
   },
+  {
+    input: `
+<Banner>
+  Escaped \\*
+</Banner>
+`,
+    blockNode: {
+      fields: {
+        blockType: 'Banner',
+        content: textToRichText('Escaped *'),
+      },
+    },
+  },
```

Comment:
A test where a character is escaped within LinkNode could be useful.

Reply:
Great idea, that test failed due to lack of recursive handling of nodes. I added a test and fixed it